nautobot/extras/forms/forms.py

```diff
@@ -982,9 +986,22 @@ class Meta:
             "has_sensitive_variables",
             "job_queues_override",
             "job_queues",
+            "default_job_queue_override",
+            "default_job_queue",
             "tags",
         ]
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        job_model = self.instance
+        self.fields["default_job_queue"] = DynamicModelChoiceField(
+            queryset=JobQueue.objects.all(),
+            required=False,
+            help_text="The default job queue to route this job to",
+            label="Default Job queue",
+        )
+        self.fields["default_job_queue"].initial = job_model.default_job_queue.pk
```

Comment:
Still needed here? I'd really hope that the declaration-time field should "just work" at this point.